### ISSUE TITLE
Build issue with Webpack

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/phone-fns#1]
Seems like when you import the module on an app thats being built out by webpack the module returns an undefined. 

So

```js
import phoneFns from 'phone-fns';

console.log(phoneFns); // => is undefined
```

However doing this

```js
import format from 'phone-fns/format';

console.log(format); // => returns the function
```

Will be digging into this for a `v0.3.1` patch.

**Comment by dhershman1** [comment]
Fixed with release of [v0.3.1](https://github.com/dhershman1/phone-fns/releases/tag/v0.3.1)

**Comment by dhershman1** [comment]
Doesn't look like the patch worked as intended because I am a dumb.
